src/Adyen/Util/HmacSignature.php

```diff
@@ -28,6 +29,26 @@ public function validateHMAC(string $hmacKey, string $hmacSign, string $webhook)
         ));
         return hash_equals($expectedSign, $hmacKey);
     }
+    /**
+     * @param string $hmacKey Can be found in Customer Area
+     * @param string $hmacSign Can be found in the Webhook headers
+     * @param string $webhook The response from Adyen
+     * @return bool
+     * @throws AdyenException
+     */
+    public function validateHMACSignature(string $hmacKey, string $hmacSign, string $webhook): bool
```

Comment:
Are you not switching it up now? The code just committed should be the way forward and this method is the "old" wrong way no?

Reply:
Ah you're right it seems like I've pulled the non updated version!

Reply:
Ahhh okay, you got it right. Thought you already rebased on the contribution but you didn't :)